src/syntax/ast.rs

```diff
@@ -28,7 +31,6 @@ pub struct Pattern {
 
 #[derive(Debug, PartialEq)]
 pub enum Expression {
-    Pattern(Pattern),
     String(String),
     Number(Number),
```

Comment:
Why would we need `StringExpression` struct? What field would it hold except of a string? Where else would it be used?